tests/FakeItEasy.Tests/CompletedFakeObjectCallExtensionsTests.cs

```diff
@@ -1,62 +0,0 @@
-namespace FakeItEasy.Tests
-{
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Linq.Expressions;
-    using FakeItEasy.Core;
-    using FakeItEasy.Expressions;
-    using FakeItEasy.Tests.TestHelpers;
-    using FluentAssertions;
-    using Xunit;
-
-    public class CompletedFakeObjectCallExtensionsTests : ConfigurableServiceLocatorTestBase

```

Comment:
Yeah. I actually worked for a while trying to convert existing tests. In some places, like for `FakeConfigurationManagerTests`, I succeeded. Elsewhere, not so much. I cannot remember the difficulty here. Maybe I just thought the tests were "too thin" to bother with. For `StartConfigurationTests`, I tried for a while, and failed. Truth be told, I couldn't even understand them.